### DISCUSSION TITLE
seqcli query, return more then 1024 rows

### DISCUSSION THREAD

**Discussion by ruant** [datalust/seq-tickets#1586] [Help] [answered]
I'm querying a "large" section of our logs to extract specific properties from the events.
But `query` is limited to 1024 rows. Is there a way to increase this?

I see that `search` has the `--count` option, but this is missing on `query`.
And since I'm trying to get specific properties on the event I'm not able to use search. Not that I've managed at least.

**Comment by nblumhardt** [comment, marked as answer]
HI @ruant - including a `limit` clause in the query itself should do what you're after.

E.g.

```sql
select @Message from stream limit 10000
```

Hope this helps,
Nick

**Reply by ruant** [reply]
Of course 🤦‍♂️
I got hung up on the cli options.

Works like a charm!
Cheers @nblumhardt!

**Reply by nblumhardt** [reply]
Great, thanks @ruant 👍
